apps/webapp/src/state/send.ts

```diff
@@ -107,6 +130,19 @@ const planWitnessBuildBroadcast = async ({ amount, recipient, selection, memo }:
 
   const { plan } = await viewClient.transactionPlanner(req);
 
+  return plan;
+};
+
+const planWitnessBuildBroadcast = async ({
```

Comment:
This works exactly as before, even throwing the same errors as before. I've only extracted a `getPlan` function above.